feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.spec.ts

```diff
@@ -516,4 +517,32 @@ describe('ConfigAddToCartButtonComponent', () => {
       });
     });
   });
+  describe('Accessibility', () => {
+    it("should contain add to cart button element with 'aria-label' attribute that contains prices of the configuration", () => {
+      CommonConfiguratorTestUtilsService.expectElementContainsA11y(
+        expect,
+        htmlElem,
+        'button',
+        undefined,
+        0,
+        'aria-label',
+        component.getButtonResourceKey(
+          mockRouterData,
+          mockProductConfiguration
+        ) +
+          ' ' +
+          'configurator.a11y.addToCartPrices' +
+          ' ' +
+          'basePrice:' +
+          mockProductConfiguration.priceSummary?.basePrice?.formattedValue +
+          ' ' +
+          'selectedOptions:' +
+          mockProductConfiguration.priceSummary?.selectedOptions
+            ?.formattedValue +
+          ' ' +
+          'totalPrice:' +
+          mockProductConfiguration.priceSummary?.currentTotal?.formattedValue
```

Comment:
in my opinion it could be m ore readable to extract constant and make a template string out of it 
`{'Base Price {{basePrice}}, Selected Options {{selectedOptions}}, Total Price {{totalPrice}}'`